### PR TITLE
Initial structure

### PR DESCRIPTION
**Pull request by markysoft**
https://eaflood.atlassian.net/browse/FCEP-6

Initial structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,197 @@
-# ffc-ce-web
-Future Farming and Countryside programme calculcation engine web
+# FFC Calculation Engine
+
+Boilerplate text.
```
Comment (Matthew-Collins-Defra) [inline README.md]: Maybe "Boilerplate" should be replaced with something like
This is a demo application for the FFC Calculation Engine

File: helm/ffc-ce-web/jenkins-aws.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+environment: production
+service:
+  port: 443
+  type: NodePort
+container:
+  allowPrivilegeEscalation: false
+  imagePullPolicy: Always
+  paymentUrl: http://ffc-ce-payment-orchestrator.ffc-ce-payment-orchestrator-master
```
Comment (Matthew-Collins-Defra) [inline helm/ffc-ce-web/jenkins-aws.yaml:8]: This will need to change when the backend github project is up and running

**Review by Matthew-Collins-Defra — APPROVED** [review]
Looks good to me. A few minor things, but this is a lot of stuff to check so I'm bound to have missed something.
The jenkins job runs and deploys to a PR end point that works. The pods look fine on k8s, the tests run locally and in containers. I can run it locally in a container, and all the watch scripts work. So we've definately got enough to be getting on with.

File: helm/ffc-ce-web/jenkins-aws.yaml (hunk 2 above)
Comment (markysoft) [reply helm/ffc-ce-web/jenkins-aws.yaml:8]: true, once the back end service is merged the old deployment can be deleted and the url updated. If the back end happens to get merged before I finish these changes I'll update it to save time

**Pull request merged**